packages/common/src/feedback/edit.ts

```diff
@@ -0,0 +1,68 @@
+import { IHubFeedback, IModel, getUniqueSlug, setDiscussableKeyword } from "..";
+import { getModel, updateModel } from "../models";
+import { PropertyMapper } from "../core/_internal/PropertyMapper";
+import { setEntityStatusKeyword } from "../utils/internal/setEntityStatusKeyword";
+import { getPropertyMap } from "./_internal/getPropertyMap";
+import { computeProps } from "./_internal/computeProps";
+import { IUserRequestOptions } from "@esri/arcgis-rest-auth";
+import { IUserItemOptions, removeItem } from "@esri/arcgis-rest-portal";
+
+/**
+ * @private
+ * Update a Hub Feedback obejct
+ * @param feedback the feedback to update
+ * @param requestOptions user request options
+ * @returns promise that resolves a IHubFeedback
+ */
+export async function updateFeedback(
+  feedback: IHubFeedback,
+  requestOptions: IUserRequestOptions
+): Promise<IHubFeedback> {
+  // verify that the slug is unique, excluding the current feedback
+  feedback.slug = await getUniqueSlug(
+    { slug: feedback.slug, existingId: feedback.id },
+    requestOptions
+  );
+  // update the status keyword
+  feedback.typeKeywords = setEntityStatusKeyword(
+    feedback.typeKeywords,
+    feedback.status
+  );
+  feedback.typeKeywords = setDiscussableKeyword(
```

Comment:
We'll need a similar `setDisplayModeKeyword` util here that either adds or removes `hubMapSurvey` based on the value of `feedback.displayMode` here

```ts
feedback.typeKeywords = setDisplayModeKeyword(
  feedback.typeKeywords,
  feedback.displayMode
);
```